stores/stores.go

```diff
@@ -506,3 +506,13 @@ var ExampleFlatTree = sops.Tree{
 		},
 	},
 }
+
+// HasSopsTopLevelKey returns true if the given branch has a top-level key called "sops".
+func HasSopsTopLevelKey(branch sops.TreeBranch) bool {
+	for _, b := range branch {
+		if b.Key == "sops" {
```

Comment:
Not worth it for this PR, but thoughts on adding a constant for `"sops"`? 

https://github.com/search?q=repo:getsops/sops+sops%22+language:Go&type=code

Reply:
Yeah, that would be a very good idea!

Reply:
#1398